client/packages/admin/src/index.ts

```diff
@@ -556,6 +558,142 @@ class Auth {
   }
 }
 
+type UploadMetadata = { contentType?: string } & Record<string, any>;
+type StorageFile = {
+  key: string;
+  name: string;
+  size: number;
+  etag: string;
+  last_modified: number;
```

Comment:
nit: should we make this `lastModified`?

Reply:
this comes directly from the server, and it seems like the pattern used in the rest of the codebase is to keep server API responses in snakecase 🤷 happy to change, but we should probably all align on what the convention should be

Reply:
SGTM! Yeah we have not been consistent. Let's keep snake_case. Eventually we can inc a version, and write some middleware that converts based on the client asking.